polaris/management/commands/testnet.py

```diff
@@ -320,3 +417,162 @@ def get_balance(self, code, issuer_public_key, json) -> Optional[str]:
                 and balance_obj.get("asset_issuer") == issuer_public_key
             ):
                 return balance_obj["balance"]
+
+    def create_trustline(self, code :str, asset_issuer_public_key: str, distribution: Keypair):
+        accounts = self.get_or_create_accounts([distribution])
+        balance = self.get_balance(code, asset_issuer_public_key, accounts[distribution.public_key])
+        if balance:
+            print(f"\nTrustline to {code}:{asset_issuer_public_key} already exists")
+            return
+        print(f"\nCreating trustline to {code}:{asset_issuer_public_key}")
+        tb = TransactionBuilder(
+            self.server.load_account(distribution.public_key),
+            base_fee=settings.MAX_TRANSACTION_FEE_STROOPS
+            or self.server.fetch_base_fee(),
+            network_passphrase="Test SDF Network ; September 2015",
+        )
+        tb.append_change_trust_op(
+            asset=StellarSdkAsset(code=code, issuer=asset_issuer_public_key),
+            source=distribution.public_key,
+        )
+        envelope = tb.set_timeout(30).build()
+        envelope.sign(distribution)
+        try:
+            self.server.submit_transaction(envelope)
+        except BaseHorizonError as e:
+            print(
+                f"Failed to create trustline "
+                f"Result codes: {e.extras.get('result_codes')}"
+            )
+        else:
+            print(f"Success! Trustline to {code}:{asset_issuer_public_key} created!")
```

Comment:
We have an `add_balance()` function, can we refactor this and that function so we're not duplicating code?